VisualStudioPlugin/IBacktestObserver.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace QuantConnect.VisualStudioPlugin
+{
+    /// <summary>
+    /// Backtest observer interface. Used to notify backtest status executed from the IDE.
+    /// </summary>
+    internal interface IBacktestObserver
+    {
+        void NewBacktest(int projectId, Api.Backtest backtestStatus);
+        void UpdateBacktest(int projectId, Api.Backtest backtestStatus);
+        void BacktestFinished(int projectId, Api.Backtest backtestStatus);
```

Comment:
this may be me being nit picky :D

I love the 'eventy' sound of `BacktestFinished` -- it matches well with 'observer' -- what are your thoughts on renaming `NewBacktest` to something like `BacktestCreated` or `BacktestStarted`, and likewise for `UpdateBacktest` to `BacktestStatusUpdated`?

Another options is simply `Started`, `Updated`, and `Finished` since the 'backest' portion is already implied via the type name and the parameter of type `Api.Backtest`.

Reply:
Your right! good suggestion, I'll use Backtest{event} format 👍